apps/core/scenes/search/PlacePicker/PlacePickerRenderer.js

```diff
@@ -23,15 +25,24 @@ export default class PlacePickerRenderer extends React.Component<Props> {
 
   render() {
     return (
-      <QueryRenderer
-        query={graphql`
-          query PlacePickerRendererQuery($input: LocationsByTermInput!) {
-            ...PlacePickerContent_locations @arguments(input: $input)
-          }
-        `}
-        variables={this.getInputVariable()}
-        render={this.renderInner}
-      />
+      <View style={styles.container}>
```

Comment:
I am not sold on moving the styles here. I think this component should only deal with rendering the query. Styling the container can be done in the Content component as it was before.

Reply:
But we are not able to style spinner inside of the Content component:

<img width="386" alt="screenshot 2019-02-13 at 13 35 32" src="https://user-images.githubusercontent.com/858321/52712239-47561580-2f94-11e9-831c-694d50244c16.png">

Reply:
I think we can improve our QueryRenderer to be able to render the custom component for loading. What do you think?
cc @tbergq

Reply:
I don't think our issue is with the loader but with our modal.

Reply:
The picker still uses old modal implementation. After `universal-components` version update we will be able to replace it with new version.
So we can probably keep this style for now to make picker look better. And we will remove it after update.
Maybe just add some `todo` note to styles to keep this in mind.

Reply:
Sounds good, `@TODO` added